### PR TITLE
Update context to reflect model 2.0

### PR DESCRIPTION
**Pull request by birkland**
Contains all suggested additions and deletion to entities and attributes.

We may wish to remove `Workflow`, `Identifier` and associated fields.  Those are indented in the context.

### CODE REVIEW

File: src/main/resources/context-2.0.jsonld [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    "repository": {"@id": "pass:repository", "@type": "@id"},
+    "repositoryCopy": {"@id": "pass:repositoryCopy", "@type": "@id"},
+    "repositories": {"@id": "pass:repositories", "@container": "@set", "@type": "@id"},
+    "roles": {"@id": "pass:roles", "@container": "@set"},
+    "source": {"@id": "pass:source"},
+    "startDate": {"@id": "pass:startDate", "@type": "xsd:dateTime"},
+      "step": {"@id": "pass:step"},
+      "steps": {"@id": "pass:steps"},
+    "submission": {"@id": "pass:submission", "@type": "@id"},
+    "submitted": {"@id": "pass:submitted", "@type:": "@id"},
```
Comment (karenhanson) [inline src/main/resources/context-2.0.jsonld]: `submitted` is an xsd:boolean

File: src/main/resources/context-2.0.jsonld [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    "policy": {"@id": "pass:policy", "@type": "@id"},
+    "policyUrl": {"@id": "pass:policyUrl", "@type": "@id"},
+    "primaryFunder": {"@id": "pass:primaryFunder", "@type": "@id"}, 
+    "projectName": {"@id": "pass:projectName"},
+    "publisher": {"@id": "pass:publisher", "@type": "@id"},
+    "publication": {"@id": "pass:publication", "@type": "@id"},   
+    "repository": {"@id": "pass:repository", "@type": "@id"},
+    "repositoryCopy": {"@id": "pass:repositoryCopy", "@type": "@id"},
+    "repositories": {"@id": "pass:repositories", "@container": "@set", "@type": "@id"},
+    "roles": {"@id": "pass:roles", "@container": "@set"},
```
Comment (karenhanson) [inline src/main/resources/context-2.0.jsonld]: there is also:
`"role": {"@id": "pass:role"},`
at `User.role`

Comment (birkland) [reply src/main/resources/context-2.0.jsonld]: There is a suggestion on the table to just re-use `roles`, see:
https://github.com/OA-PASS/pass-data-model/pull/24#discussion_r182490197

Comment (cameronblandford) [reply src/main/resources/context-2.0.jsonld]: UI team is okay with removing workflows, we don't use them at all

**Pull request merged**